spec/services/npq/amend_participant_cohort_spec.rb

```diff
@@ -62,7 +62,7 @@
     context "when lead provider has no contract for the cohort and course" do
       before { npq_contract.update!(npq_course: create(:npq_specialist_course)) }
 
-      it "is invalid and returns an error message", :flakey_test do
```

Comment:
I was wondering when/how this was fixed before as the error was recent?

Reply:
Hi @cwrw I had included a fix in one of my PRs

https://github.com/DFE-Digital/early-careers-framework/pull/4127/files#diff-f02c377365ed22cfad1ca78ccf9080b60890094da9740e14dca805371631e338